sportladder.html

```diff
@@ -0,0 +1,114 @@
+    compulsory tricks of increasing difficulty. The contestant who completes the most tricks most consistently
+    wins. See below for trick list and rules.</p>
+
+    <p>This is a non-championship event. Anyone can participate by registering at the CANYA table before 12:30PM. See <a href='schedule.html'>schedule</a>.</p>
+
+    <h1>Trick list</h1>
+    <ol>
+        <li>Double or nothing</li>
+        <li>Ninja Vanish</li>
+        <li>1.0 hook</li>
+        <li>Brent Stole</li>
+        <li>Green Triangle suicide (any entry into GT)</li>
+        <li>1.5 hook</li>
+        <li>One hand bind (throw and bind - any variation without using NTH)</li>
+        <li>DNA bind (yoyo horizontal with DNA pointing up)</li>
+        <li>2.0 hook</li>
+        <li>One hand regeneration and bind (throw, bind, regenerate, bind - any variation without using NTH)</li>
+        <li>2.5 hook</li>
+        <li>3.0 hook</li>
+    </ol>
+    <p>(NTH = Non throw hand)</p>
+
+    <h1>Format</h1>
+    <ul>
+        <li>For each trick, each contestant has three attempts:
+        <ul>
+            <li>Landing on the first attempt is worth <strong>5</strong> points</li>
+            <li>Landing on the second attempt is worth <strong>2</strong> points</li>
+            <li>Landing on the third attempt is worth <strong>1</strong> point</li>
+            <li>Missing all three attempts results in a <strong>zero</strong></li>
+        </ul>
+        </li>
+
+        <li>After a contestant scores a <strong>zero</strong>, the contestant cannot remain on stage. However this does not mean a loss, depending on final score.</li>
+
+        <li>Ladder continues until either all contestants are out, or all remaining contestants have completed the ladder. The contestant
+        with the highest score wins.</li>
+
+        <li>Tie breaker: Hook ladder sudden death. All remaining contestants take turns to attempt hooks starting from
+        1.0, 1.5, 2.0, 2.5, 3.0, and so on. </li>
+    </ul>
+
+    <h1>Rules</h1>
+    <ul>
+        <li>Showmanship does not count. If variations are allowed, the trick may consist of nothing more than the required element(s).</li>
+
+        <li>Keep in mind the number of <strong>ATTEMPTS</strong> is not the number of <strong>THROWS</strong>. A contestant may, intentionally or otherwise,
+        attempt more than once with one throw.</li>
+
+        <li>Contestants can use whichever yoyo is optimal for each trick. Contestants are allowed to switch, adjust, or maintain yoyos between attempts.</li>
+
+        <li>After successfully completing the required trick element, misses are tolerated. Missed binds at the end are tolerated unless
+        the bind is a required element.</li>
+    </ul>
+
+    </div>
+    
+    <div id="sponsor-footer">
+    <h2>Sponsors</h2>
+    </div>
+    
+    <script src="sponsors.js"></script>
+    <script>
+        loadGoldSponsors("sponsor-footer", 0.6);
```

Comment:
Change sponsor scale to 0.5 (same as other pages), otherwise it's currently blocking some content